src/rulesets/asyncapi/__tests__/asyncapi2-channel-path-no-empty-parameter-declaration.ts

```diff
@@ -0,0 +1,45 @@
+import { cloneDeep } from 'lodash';
+
+import { buildTestSpectralWithAsyncApiRule } from '../../../../setupTests';
+import { Spectral } from '../../../spectral';
+import { IRunRule } from '../../../types';
+
+const ruleName = 'asyncapi2-channel-path-no-empty-parameter-declaration';
```

Comment:
ooof can we find a shorter name for this? We might trigger some UI issues like this.

Reply:
@philsturgeon Ok. I'll try and come up with shorter names.

Reply:
@philsturgeon Shorter names! Could you please check they still make sense?